inc/ThirdParty/Themes/Bridge.php

```diff
@@ -94,4 +93,18 @@ public function maybe_clear_cache( $old_value, $new_value ) {
 			rocket_clean_minify();
 		}
 	}
+
+	/**
+	 * Excludes Bridge statically generated files.
+	 *
+	 * @param array $files Array of static files to be excluded.
+	 *
+	 * @return array Excluded files with Bridge entries to be excluded.
+	 */
+	public function exclude_static_file_generation( $files ) {
+		$files[] = 'wp-content/themes/bridge/js/default_dynamic_callback.php';
+		$files[] = 'wp-content/themes/bridge/css/style_dynamic_callback.php';
```

Comment:
I'd suggest to get the path more dynamically here, since the `wp-content` folder can be a different name.

There is an example here: https://github.com/wp-media/wp-rocket/blob/e2c615669d775508771c0bcfa40eef9d69068265/inc/ThirdParty/Themes/Avada.php#L133